tests/www/apply/test_process.py

```diff
@@ -2235,7 +2236,22 @@ def test_refuse_jobapplication_geiq_reasons(client, reason):
     }
 
 
-def test_details_for_prescriber_geiq(client):
+def test_details_for_prescriber_not_can_have_prior_actions(client):
+    kind = random.choice([kind for kind in CompanyKind.values if kind != CompanyKind.GEIQ])
+    job_application = JobApplicationFactory(
+        sent_by_authorized_prescriber_organisation=True,
+        state=JobApplicationWorkflow.STATE_PROCESSING,
+        to_company__kind=kind,
+    )
+    prescriber = job_application.sender_prescriber_organization.members.first()
+    client.force_login(prescriber)
```

Comment:
Ça devrait être équivalent, sauf si tu voulais avoir un autre membre de l'organisation ? Mais donc peut-être l'expliciter ?
```suggestion
    client.force_login(job_application.sender)
```